rest/pagination.py

```diff
@@ -0,0 +1,16 @@
+"""
+Pagination classes for Lore's REST API.
+"""
+
+from __future__ import unicode_literals
+
+from rest_framework.pagination import PageNumberPagination
+
+
+class LorePagination(PageNumberPagination):
+    """
+    Pagination class for Lore's REST API.
+    """
+    page_size = 20
+    page_size_query_param = 'page_size'
+    max_page_size = 1000
```

Comment:
20 is the default page size, 1000 is just some number I considered to be big enough for one request, a guess